22_CppCompat/app_resources/tgmathTest.comp.hlsl

```diff
@@ -0,0 +1,16 @@
+//// Copyright (C) 2023-2024 - DevSH Graphics Programming Sp. z O.O.
+//// This file is part of the "Nabla Engine".
+//// For conditions of distribution and use, see copyright notice in nabla.h
+#pragma shader_stage(compute)
+
+#include "common.hlsl"
+
+[[vk::binding(0, 0)]] RWStructuredBuffer<TgmathIntputTestValues> inputTestValues;
+[[vk::binding(1, 0)]] RWStructuredBuffer<TgmathTestValues> outputTestValues;
+
+[numthreads(16, 1, 1)]
+void main(uint3 invocationID : SV_DispatchThreadID)
+{
+    if(invocationID.x == 0)
+        outputTestValues[0].fillTestValues(inputTestValues[0]);
```

Comment:
you could try more values than just 1 set

Reply:
there's something I don't get, you do 16 dispatches of 16 wide workgroups (should be at least 64 workgroup size if not 256 btw), so you have 16*16=256 invocations and I can see you in the Tester init code actually use random distributions to provide multiple test inputs.

Then why is only the first invocation of every workgroup (there are 16) only reading from and writing to the first global memory location?

Reply:
corrected, now it is a single dispatch of 256 wide workgroup where only the first invocation does anything. I wanted this test to be convenient not efficient